db_client/alembic/versions/0035_add_corpus_text_and_image.py

```diff
@@ -0,0 +1,86 @@
+"""add corpus text and image
+
+Revision ID: 0035
+Revises: 0034
+Create Date: 2024-04-30 11:22:43.893840
+
+"""
+
+from typing import Optional
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import Session
+
+from db_client.models import ORGANISATION_CCLW, ORGANISATION_UNFCCC
+
+# revision identifiers, used by Alembic.
+revision = "0035"
+down_revision = "0034"
+branch_labels = None
+depends_on = None
+
+
+Base = automap_base()
+
+
+CCLW_TEXT = """
+        <p>
+          The summary of this document was written by researchers at the <a href="http://lse.ac.uk/grantham" target=""> Grantham Research Institute </a> . 
+          If you want to use this summary, please check <a href="https://www.lse.ac.uk/granthaminstitute/cclw-terms-and-conditions" target=""> terms of use </a> for citation and licensing of third party data.
```

Comment:
Can you add `target="_blank"` to these links please? This will mean they open in a new tab rather than navigate away from the site (see here https://www.w3schools.com/tags/att_a_target.asp)

Reply:
Thanks @PatFawbertMills :+1: